server/spec/test_external_resources_file.rb

```diff
@@ -50,17 +53,17 @@ def test_list_resources_top_empty_string
 
   def test_list_resources_images
     parsed_file_list = JSON.parse(@erf.get_resource "image")
-    assert_equal 8,parsed_file_list.length,"list resources in images directory"
+    assert_equal 9,parsed_file_list.length,"list resources in images directory"
```

Comment:
There is no newly added image file in this checkin. Not sure why this number need to be increased.